Tests/geometry/regions.lisp

```diff
@@ -300,18 +300,19 @@
          (some-el (list el1 el2 el3 el4))
```

Comment:
> Tests for belinging with a difference of 1 will likely fail because we factor an additional epsilon for ellipse tests. Without that clx rendering of rotated ellipses doesn't work correctly.

"belinging" seems like a typo.

But also, please add a FIXME and comment like this to the test in question.

Reply:
fixed